integration/test_rdb_indexes_restored_percent.py

```
@@ -0,0 +1,174 @@
+                try:
+                    info = monitor_client.info("search")
+                    if info.get("search_rdb_restore_in_progress", 0):
+                        pct = float(info["search_rdb_indexes_restored_percent"])
+                        observed_percents.append(pct)
+                        print(f"[Standalone][Monitor] restore_in_progress=1, indexes_restored_percent={pct:.2f}%")
+                        # Verify the metric never exceeds 100%
+                        if pct > 100.0:
+                            monitor_error[0] = f"Metric exceeded 100%: {pct}"
+                            return
+                except Exception:
+                    pass
```

Ruff (0.16.0)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)